src/__tests__/functions.test.ts

```diff
@@ -4,6 +4,7 @@ import { IRuleResult, Rule, RuleFunction } from '../types';
 const applyRuleToObject = async (r: Rule, o: object): Promise<IRuleResult[]> => {
   const s = new Spectral();
   s.addRules({
+    // @ts-ignore
```

Comment:
This does not seem to be required.

![image](https://user-images.githubusercontent.com/1416224/57933949-24cde400-7884-11e9-946e-0e53e6dc3569.png)
```suggestion
```

Reply:
I still see the comment here, we can remove this right?